pkgs/tools/admin/daemontools/default.nix

```diff
@@ -1,5 +1,11 @@
-{ fetchurl, bash, glibc, lib, stdenv }:
+{ fetchurl, bash, glibc, lib, stdenv, installShellFiles }:
 
+let
+  man-pages = fetchurl {
+    url = "https://salsa.debian.org/debian/daemontools/-/archive/debian/1%250.76-8/daemontools-debian-1%250.76-8.tar.gz?path=debian/daemontools-man";
```

Comment:
I was thinking about this earlier but didn't have a good solution: how do we make sure those match the version of daemontools? We probably just need to watchout when updating.

Reply:
Given that the last release of daemontools was in 2001, I don't think we have to care too much. Watching it manually is definitely sufficient.